airbyte-webapp/src/pages/connector-builder/ConnectorBuilderPage.module.scss

```diff
@@ -0,0 +1,14 @@
+@use "scss/colors";
+
+.leftPanel {
+  overflow: hidden;
+}
+
+.rightPanel {
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+}
```

Comment:
Would be nice to use the standard `$border-radius-md` here

Reply:
Good catch! Fixed